tests/test_lab_dev/test_circuits.py

```diff
@@ -193,9 +193,9 @@ def test_repr(self):
 
         circ2 = Circuit([vac012, s01, bs01, bs12, cc, n12.dual])
         r2 = ""
-        r2 += "\nmode 0:     ◖Vac◗──S(0.0,2.0)──╭•──────────────────────────CC──|3)="
-        r2 += "\nmode 1:     ◖Vac◗──S(1.0,3.0)──╰BS(0.0,0.0)──╭•────────────CC──|3)="
-        r2 += "\nmode 2:     ◖Vac◗────────────────────────────╰BS(0.0,0.0)──────────"
+        r2 += "\nmode 0:     ◖Vac◗──S(0.0,2.0)──╭•──────────────────────────CC──|3)=(3,3)"
+        r2 += "\nmode 1:     ◖Vac◗──S(1.0,3.0)──╰BS(0.0,0.0)──╭•────────────CC──|3)=(3,3)"
+        r2 += "\nmode 2:     ◖Vac◗────────────────────────────╰BS(0.0,0.0)───────────────"
```

Comment:
I'm not sure I understand the notation. What is (3,3) ?

Reply:
`(3,3)` comes from the parameter set so this is just a byproduct of the param set of `.dual` actually being populated. To see this if you just do `repr(Circuit([n12]))` you get 
```
mode 0:     ◖3◗(3,3)
mode 1:     ◖3◗(3,3)
```
I think `|3)=(3,3)` does look a bit funky and unintuitive but this would require updating the Circuit repr